installer/templates/phx_live/controllers/page_controller.ex

```diff
@@ -0,0 +1,7 @@
+defmodule <%= web_namespace %>.PageController do
+  use <%= web_namespace %>, :controller
+
+  def index(conn, _params) do
+    render(conn, "index.html")
```

Comment:
Moved away from 100% liveview, and instead did an embedded example within the `index.html.eex`